index.html

```diff
@@ -16,58 +13,251 @@
   <body>
 
   <header>
-       <nav class="navbar navbar-expand-md navbar-light">
-        <div class="container">
-        <a href="#" class="navbar-brand">My Logo</a>
-        <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbarCollapse">
-            <span class="navbar-toggler-icon"></span>
-        </button>
-
-        <div class="collapse navbar-collapse justify-content-between" id="navbarCollapse">
-            <div class="navbar-nav"> </div>
-            <div class="navbar-nav">
-                <a href="#" class="nav-item nav-link active">Porfolio</a>
-                <a href="#" class="nav-item nav-link">About</a>
-                <a href="#" class="nav-item nav-link">Contact</a>
-            </div>
-        </div>
-        </div>
+    <nav class="navbar">
```

Comment:
- [ ] this section  should be enclosed in section tag

Reply:
Hi ,
Can you tell me what's wrong with this? I don't really understand.  In screenshot, you will see what I did. 

![Screenshots](https://user-images.githubusercontent.com/6615394/126033701-b36b39c6-bdae-4b20-93da-814a0706d65c.PNG)